### ISSUE TITLE
[Bug]: Add Teams with Labels but no Quotas

### ISSUE THREAD

**Issue by allamand** [aws-ia/terraform-aws-eks-blueprints#658]
### Welcome to Amazon EKS Blueprints!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/aws-ia/terraform-aws-eks-blueprints/issues) and didn't find any.

### Amazon EKS Blueprints Release version

4.0.7

### What is your environment, configuration and the example used?

I wanted to create a namespace without quotas limitation as My add is not yet compatible with this (no ressources set)

```
    test = {
      "labels" = {
        "elbv2.k8s.aws/pod-readiness-gate-inject" = "enabled",
        "appName"     = "test",
        "projectName" = "test",
        "environment" = "dev",
      }

      users         = [data.aws_caller_identity.current.arn]
    }
```

### What did you do and What did you see instead?

I wanted the namespace to be created with the labels I provided.

Instead I got thoses errors:

```
Error: Invalid index
│ 
│   on .terraform/modules/eks_blueprints/modules/aws-eks-teams/main.tf line 23, in resource "kubernetes_resource_quota" "team_compute_quota":
│   23:       "requests.cpu"    = each.value["quota"]["requests.cpu"]
│     ├────────────────
│     │ each.value is object with 2 attributes
│ 
│ The given key does not identify an element in this collection value.
╵
╷
│ Error: Invalid index
│ 
│   on .terraform/modules/eks_blueprints/modules/aws-eks-teams/main.tf line 24, in resource "kubernetes_resource_quota" "team_compute_quota":
│   24:       "requests.memory" = each.value["quota"]["requests.memory"]
│     ├────────────────
│     │ each.value is object with 2 attributes
│ 
│ The given key does not identify an element in this collection value.
...

```
same for every field of the quotas.

**Comment by Hokwang** [comment]
this issue is related with https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/578

**Comment by bryantbiggs** [comment]
Closes in #678
